### DISCUSSION TITLE
Demuxed audio when creating a new FAST channel?

### DISCUSSION THREAD

**Discussion by anderswassen** [EyevinnOSC/community#33] [Q&A] [answered]
Hi!

When I created a new FAST channel using your awesome FAST channel engine, I realized that the Demuxed audio switch had to be turned off. Can you please verify that that is the case and explain why?

Keep up the good work!

**Comment by birme** [comment, marked as answer]
Hi!

As a maintainer of the FAST channel engine I would like to first thank you for trying it out!

I recognize that the documentation can be more clear on this part. What the option `Demuxed audio` does is that it instructs the engine what it can expect for type of VOD files. There are two types:
- a) Audio and Video is contained in the same video package
- b) Audio and Video is kept in different files and video packages (demuxed)
The example playlist that is provided in the tutorial contains VOD files of type (a) and that is why you need to set the `Demuxed audio` option to false.

Hope that answered your question!

/Jonas
